packages/tpetra/core/src/Tpetra_FECrsGraph_decl.hpp

```diff
@@ -128,11 +128,11 @@ namespace Tpetra {
     /// \param ownedPlusSharedToOwnedimporter [in] Optional importer between the ownedMap and ownedPlusSharedMap
     ///   This will be calculated by FECrsGraph if it is not provided
     ///
-    /// \param domainMap [in] Optional domainMap for the owned graph.  If this is not provided, then ownedMap
-    ///   will be used for the domainMap in the call to endFill()
+    /// \param ownedDomainMap [in] Optional domain map for both owned and owned+shared graphs. If this is not provided,
```

Comment:
Ah, good catch. I was updating the comments to stress the fact that that map was for the owned graph only, but I must have messed up with copy/paste from another ctor. I will go over the comments again, fix this, and make sure there are no other mistakes.

Reply:
Ah, sorry, this is the already existing ctor. Indeed, the old ctor was using this map for both owned and owned+shared. I will change the input var name so that it does not imply owned or owned+shared.